perf-tests/clusterloader2/pkg/util/multi-tenancy.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	apimachineryvalidation "k8s.io/apimachinery/pkg/api/validation"
+)
+
+func GetTenant() string {
```

Comment:
fyi, the two initial test tenant for POC is named as "arktos" and "zeta"

Reply:
The specific tenant names won't impact this PR, right?  The test script will fetch whatever tenant name set in environment variable from command line.